Content.Client/_RMC14/Explosion/CMExplosionShockWaveOverlay.cs

```diff
@@ -0,0 +1,92 @@
+using Content.Shared._RMC14.Explosion.Components;
+using Robust.Client.Graphics;
+using Robust.Shared.Enums;
+using Robust.Shared.Prototypes;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Content.Client._RMC14.Explosion
```

Comment:
`namespace Content.Client._RMC14.Explosion;`

you can put a semicolon here and drop a whole line of indentation across the entire file